### PR TITLE
Layout image-annotation-app with dash-bootstrap-components and reduce css

### PR DESCRIPTION
**Pull request by surchs**
Issue for app: #501 

# App pull request
- [ ] This is a new app
- [x] I am improving an existing app (redesigns/code "makeovers")

## About

- Playground deployment URL: https://dash-playground.plotly.host/dash-image-annotation/
- Current gallery app URL: https://dash-gallery.plotly.host/dash-image-annotation/

## Workflow

- [x] I have created a branch in the appropriate monorepo, and the
  elements necessary for successful deployment are in place.
- [x] If the app is a redesigned and/or restyled version of an
  existing gallery app, I've summarized the changes requested in the
  appropriate Streambed issue and confirm that they have been applied.
- [ ] If the app is on the Dash Gallery portal, I have added a link to
  the GitHub repository for the source code in the portal description.
- [ ] If the app is a reimplementation of a Python gallery app for the
  DashR gallery, the app in this PR mimics, as closely as possible,
  the style and functionality of the existing app.=
- [ ] I have removed *all Google Analytics code* from the app's
  `assets/` folder.

## The pre-review review

I have addressed all of the following questions:

- [x] Does everything in my code serve some purpose? (I have removed
  any dead and/or irrelevant code.)
- [x] Does everything in my code have a clear purpose? (My code is
  readable and, where it isn't, it has been commented appropriately.)]
- [x] Am I reinventing the wheel? (I have used appropriate packages to
  lessen the volume of code that needs to be maintained.)

### CODE REVIEW

File: apps/dash-image-annotation/app.py [hunk 1]
```diff
@@ -185,165 +181,236 @@ def shape_data_remove_timestamp(shape):
...
                     figure=fig,
                     config={"modeBarButtonsToAdd": ["drawrect", "eraseshape"]},
+                )
+            ]
+        ),
+        dbc.CardFooter(
+            [
+                dcc.Markdown(
+                    "To annotate the above image, select an appropriate label on the right and then draw a "
+                    "rectangle with your cursor around the area of the image you wish to annotate\n\n"
```
Comment (nicholas-esterer) [inline apps/dash-image-annotation/app.py:233]:
```suggestion
                    "rectangle with your cursor around the area of the image you wish to annotate.\n\n"
```

**Review by nicholas-esterer — COMMENTED** [review]
Just a small punctuation suggestion. Also for some reason I thought it was possible at some point to change the annotation type in the table (say change "tree" to "road") but it doesn't seem the old one does this either. In any case the possible entries are initialized when initializing the table (see lines 277-285). So I'm wondering if this went away at some point or just never worked and was forgotten about, haha.
Also the timestamp table is missing, but maybe you've decided to let go of that, which is cool. Just wanted to make sure it wasn't forgotten :).
Otherwise, looks really good :+1:, must more consistent than what we had before.

**Comment by surchs** [comment]
Thanks for the suggestion! Yeah, I also noticed that the table dropdown didn't seem to allow changing the label. I think that didn't work in the first version I started working on. But would probably be a good idea to get to work. Maybe for another PR?

I did get rid of the timestamp table. Since it was a separate data-table object, I had a very hard time keeping it in the same line as the annotation table  when resizing the view port. The data are still available in the .json download so I opted to remove the timestamp table to avoid strange layout breaks.

**Review by emmanuelle — APPROVED** [review]
wow impressive work! Thanks, it works out really well with `dbc`. I don't see anything to change so :dancer: !

**Pull request merged**